openbb_terminal/forecast/rwd_model.py

```diff
@@ -0,0 +1,142 @@
+# pylint: disable=too-many-arguments
+"""Random Walk with Drift Model"""
+__docformat__ = "numpy"
+
+import logging
+from typing import Any, Union, Optional, List, Tuple
+
+import warnings
+import numpy as np
+import pandas as pd
+from statsforecast.models import RandomWalkWithDrift
+from statsforecast.core import StatsForecast
+
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.rich_config import console
+from openbb_terminal.forecast import helpers
+
+
+warnings.simplefilter("ignore")
+
+logger = logging.getLogger(__name__)
+
+# pylint: disable=E1123
+
+
+@log_start_end(log=logger)
+def get_rwd_data(
+    data: Union[pd.Series, pd.DataFrame],
+    target_column: str = "close",
+    seasonal_periods: int = 7,
```

Comment:
This is never used.

Reply:
thanks! fixed in [9a1dc94](https://github.com/OpenBB-finance/OpenBBTerminal/pull/3339/commits/9a1dc9432f964d568dcafea54b8f7bd0494343f3)